DEVELOPERS.md

```diff
@@ -0,0 +1,31 @@
+# Developers Notes
+
+## Setup
+
+- Please use the `make git_hooks` to setup the pre-commit hooks
```

Comment:
Although obvious, it's probably worth noting that if we are following the steps in the README.md in this repo, ```make bootstrap``` installs the dev dependencies (including poetry) inside the virtual environment, so ```make git_hooks``` in this case needs to be run inside the activated virtual environment to run successfully.

Reply:
Fair point -- I'll reference the readme for this :)